src/pages/api/images/view.ts

```diff
@@ -9,8 +10,37 @@ export default withRouteSpec({
   methods: ["GET"],
   queryParams: z.object({
     image_id: z.string().uuid(),
+    _fake_external_image_proxy_endpoint: z.string().url().optional(),
```

Comment:
Is this still needed? What uses this query param?

Reply:
`/images/view` uses this to determine if the fake should attempt to externally proxy through an image or return the built-in bundled sample one